cupy/creation/ranges.py

```diff
@@ -105,6 +104,7 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None):
             _linspace_ufunc(start, step, ret, casting='unsafe')
 
         if endpoint:
+            # num == div + 1 > 1
```

Comment:
Just I took a moment to get what this line says.
```suggestion
            # Here num == div + 1 > 1 is ensured.
```